resources/views/overview/default.blade.php

```diff
@@ -609,57 +609,57 @@ function widget_settings(data) {
     return false;
     }
 
-    function widget_reload(id, data_type) {
-        $("#widget_body_"+id+" .bootgrid-table").bootgrid("destroy");
-        $("#widget_body_"+id+" *").off();
-        var $widget_body = $("#widget_body_"+id);
-        if ($widget_body.parent().data('settings') == 1 ) {
-            settings = 1;
-        } else {
-            settings = 0;
+    function widget_reload(id, data_type, forceDomInject = false) {
+        const $widget_body = $(`#widget_body_${id}`);
+        const $widget_bootgrid = $(`#widget_body_${id} .bootgrid-table`);
+        const settings = $widget_body.parent().data('settings') == 1 ? 1 : 0;
+
+        if (settings === 1) {
+            $(`#widget_body_${id} .bootgrid-table`).bootgrid('destroy');
+            $(`#widget_body_${id} *`).off();
         }
+
         $.ajax({
```

Comment:
Why call the ajax method if you are just going to throw it away and refresh bootgrid?

Reply:
I was doing it for the title counter. With a quick search no other widget aside from alerts seems to have a counter in the title, so I have now changed the counter to be updated with the bootgrid and skip the other request.